docs/server/python/_options.mdx

```diff
@@ -7,7 +7,7 @@
 Example:
 
 ```python
-from statsig import statsig
+from statsig import statsig, StatsigEnvironmentTier
 from statsig.statsig_options import StatsigOptions
```

Comment:
Thanks!  Lets either make this:

```
from statsig.statsig_environment_tier import StatsigEnvironmentTier
```

OR

from statsig import statsig, StatsigEnvironmentTier, StatsigOptions